dashboard/config/scripts/levels/New Game Lab Jr Project.level

```diff
@@ -214,11 +214,17 @@
     "helper_libraries": [
       "GameLabJr"
     ],
-    "contained_level_names": null
+    "contained_level_names": null,
+    "encrypted_examples": [
+
+    ],
+    "droplet_tooltips_disabled": "false",
+    "lock_zero_param_functions": "false",
+    "start_animations": "{\r\n  \"orderedKeys\": [\r\n    \"9f2f01be-42ae-4242-9b47-3476cf8f1f5c\",\r\n    \"0dba258d-a2fa-471c-92d6-42f19684746e\"\r\n  ],\r\n  \"propsByKey\": {\r\n    \"9f2f01be-42ae-4242-9b47-3476cf8f1f5c\": {\r\n      \"name\": \"cat\",\r\n      \"sourceUrl\": \"http://localhost-studio.code.org:3000/v3/animations/p7sb0a3gMNYkzRBreemCkA/9f2f01be-42ae-4242-9b47-3476cf8f1f5c.png?version=_naL8ux9jreFvP_mwIcx4QZUf9QapDlj\",\r\n      \"frameSize\": {\r\n        \"x\": 64,\r\n        \"y\": 100\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"_naL8ux9jreFvP_mwIcx4QZUf9QapDlj\"\r\n    },\r\n    \"0dba258d-a2fa-471c-92d6-42f19684746e\": {\r\n      \"name\": \"dog\",\r\n      \"sourceUrl\": \"http://localhost-studio.code.org:3000/v3/animations/p7sb0a3gMNYkzRBreemCkA/0dba258d-a2fa-471c-92d6-42f19684746e.png?version=psbxMDrISyQzlR0WaHxCcihPItagUuq2\",\r\n      \"frameSize\": {\r\n        \"x\": 100,\r\n        \"y\": 92\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"psbxMDrISyQzlR0WaHxCcihPItagUuq2\"\r\n    }\r\n  }\r\n}"
```

Comment:
you probably want to use production animation URLs instead of localhost ones.

Reply:
Thanks for catching that.